python/doc/examples/data_analysis/smoothing_mixture.ipynb

```diff
@@ -0,0 +1,524 @@
+      "text/plain": [
+       "<Figure size 720x576 with 3 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "hArray = [0.05, 0.54, 0.18]\n",
+    "nLen = len(hArray)\n",
+    "fig = pl.figure(figsize=(10, 8))\n",
+    "for i in range(nLen):\n",
+    "    ax = fig.add_subplot(2, 2, i + 1)\n",
+    "    fit = factory.build(sample, [hArray[i]])\n",
+    "    graph = fit.drawPDF()\n",
+    "    graph.setColors([\"dodgerblue3\"])\n",
+    "    graph.setLegends([\"h=%.4f\" % (hArray[i])])\n",
+    "    exact = distribution.drawPDF()\n",
+    "    curve = exact.getDrawable(0)\n",
+    "    curve.setColor(\"darkorange1\")\n",
+    "    curve.setLegend(\"Mixture\")\n",
+    "    curve.setLineStyle(\"dashed\")\n",
+    "    graph.add(curve)\n",
+    "    graph.setLegendPosition(\"topleft\")\n",
+    "    graph.setXTitle(\"X\")\n",
+    "    view = otv.View(graph, figure=fig, axes=[ax])\n",
+    "    pl.ylim(top = 0.5)  # Common y-range"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We see that when the bandwidth is too low, then the resulting kernel smoothing has many more modes than the distribution it is supposed to approximate. When the bandwidth is too large, then the approximated distribution is too smooth and has only one mode instead of the expected two modes which are in the mixture distribution. When the bandwidth is equal to 0.18, then the two modes are correctly represented."
```

Comment:
```suggestion
    "We see that when the bandwidth is too small, the resulting kernel smoothing has many more modes than the distribution it is supposed to approximate. When the bandwidth is too large, the approximated distribution is too smooth and has only one mode instead of the expected two modes which are in the mixture distribution. When the bandwidth is equal to 0.18, the two modes are correctly represented."
```